Libraries/LibCore/CIPCClientSideConnection.h

```diff
@@ -112,6 +163,18 @@ class CIPCClientSideConnection {
         return response;
     }
 
+protected:
+    struct IncomingASMessageBundle {
```

Comment:
AS?

Reply:
Sorry, I meant to turn this into a draft. As the naming comment on the summary notes, there's still some work to be done I think (and this is an example).